### ISSUE TITLE
Some amount of connections

### ISSUE THREAD

**Issue by utegental** [Trejjam/Gnome-Docker-integration#4]
On ubuntu 17.10 I see this. I'm using wi-fi currently and two ethernet ports are disconnected.
![screenshot from 2017-11-26 19-53-02](https://user-images.githubusercontent.com/431860/33240639-767813b8-d2e3-11e7-867e-17e1807b431f.png)

**Comment by trejjam** [comment]
You are right, this extension was created when all connections had own toggle. With actual version of Gnome is not helping, sorry.
